lib/index.js

```diff
@@ -80,18 +89,34 @@ function parseInput (data, queryStart, queryEnd, excludeMaster, projectName) {
   });
 
   if (!excludeMaster) {
+    AddFullLineToTop(output, 'Main', 'unstable',);
+  }
+
+  AddFullLineToBottom(output, '<= v16', 'commercial');
```

Comment:
This is Node.js specific (and time sensitive) and should not be encoded into the tooling.

Reply:
oh. the tool should stay completely technology agnostic? Got it. ok, I'll have to do some rework.